Tests/SwiftParserTest/Attributes.swift

```diff
@@ -66,4 +66,13 @@ final class AttributeTests: XCTestCase {
       """
     )
   }
+
+  func testRethrowsAttribute() {
+    AssertParse(
+      """
+      @rethrows
```

Comment:
Is `@rethrows` a valid attribute? Do we need to handle `@try` or other keywords as well?

Reply:
I see. In that case you probably want to remap `rethrows` to an identifier when consuming it, right?

Reply:
Finally did this: https://github.com/apple/swift-syntax/pull/818